src/System.Net.Primitives/src/System/Net/IPAddress.cs

```diff
@@ -614,7 +615,13 @@ public override int GetHashCode()
             {
                 if (_hashCode == 0)
                 {
-                    _hashCode = StringComparer.OrdinalIgnoreCase.GetHashCode(ToString());
+                    var hashCode = HashHelpers.Combine(HashHelpers.RandomSeed, unchecked((int)PrivateScopeId));
```

Comment:
Nit: Replace vat with int. See our coding style - if the type is not clear from the line, we should not use var.

Reply:
Got it. Done